memory_storage/src/lib.rs

```diff
@@ -25,6 +25,62 @@ impl Clone for MemoryStorage {
     }
 }
 
+// For testing (KATs in particular) we want to serialize and deserialize the storage
+#[cfg(feature = "test-utils")]
+impl MemoryStorage {
+    pub fn serialize(&self, w: &mut Vec<u8>) -> std::io::Result<usize> {
```

Comment:
Now we have two ways of (de)serializing this. Please file an issue to remove one.

Reply:
Done: https://github.com/openmls/openmls/issues/1616